src/main/asciidoc/_chapters/configuration.adoc

```diff
@@ -342,17 +342,18 @@ link:https://hadoop.apache.org/cve_list.html[CVEs] so we drop the support in new
 
 .Hadoop version support matrix for active release lines
 
-[cols="1,1*^.^", options="header"]
+[cols="1,2*^.^", options="header"]
 |===
-| | HBase-2.5.x
-|Hadoop-2.10.[0-1] | icon:times-circle[role="red"]
-|Hadoop-2.10.2+ | icon:check-circle[role="green"]
-|Hadoop-3.1.0 | icon:times-circle[role="red"]
-|Hadoop-3.1.1+ | icon:times-circle[role="red"]
-|Hadoop-3.2.[0-2] | icon:times-circle[role="red"]
-|Hadoop-3.2.3+ | icon:check-circle[role="green"]
-|Hadoop-3.3.[0-1] | icon:times-circle[role="red"]
-|Hadoop-3.3.2+ | icon:check-circle[role="green"]
+| | HBase-2.5.x | HBase-2.6.x
+|Hadoop-2.10.[0-1] | icon:times-circle[role="red"] | icon:times-circle[role="red"]
+|Hadoop-2.10.2+ | icon:check-circle[role="green"] | icon:times-circle[role="red"]
```

Comment:
I thought you wanted to keep hadoop 2.10 support for now? We still have the hadoop2 profile in branch-2.6 i believe

Reply:
Ah, copy paste error.

Let me fix.

Reply:
Fixed.